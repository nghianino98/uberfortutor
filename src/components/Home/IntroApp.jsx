import React, { Component } from 'react';

class IntroApp extends Component {
  render() {
    return (
      <div className="home--card">
          <div className="intro-app">
            <h4>Về chúng tôi</h4>
            <p><b>Tìm kiếm gia sư, lớp học thêm</b> khiến bạn trở nên quá mệt mỏi, hay 
            bạn muốn <b>trở thành gia sư</b> kiếm thêm thu nhập. Hãy để <b>Uber for tutor </b>
             giải quyết giúp bạn</p>
             <button type="button" className="btn btn-primary">
              Xem thêm
             </button>
          </div>
      </div>
    );
  }

}

export default IntroApp;