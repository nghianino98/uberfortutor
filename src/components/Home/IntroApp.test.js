import React from 'react';
import renderer from 'react-test-renderer';
import IntroApp from './IntroApp';

test('test snapShot render component IntroApp', () => {
  const component = renderer.create(
    <IntroApp />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
}); 