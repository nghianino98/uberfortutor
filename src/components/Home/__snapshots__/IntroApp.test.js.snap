// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapShot render component IntroApp 1`] = `
<div
  className="home--card"
>
  <div
    className="intro-app"
  >
    <h4>
      Về chúng tôi
    </h4>
    <p>
      <b>
        Tìm kiếm gia sư, lớp học thêm
      </b>
       khiến bạn trở nên quá mệt mỏi, hay bạn muốn 
      <b>
        trở thành gia sư
      </b>
       kiếm thêm thu nhập. Hãy để 
      <b>
        Uber for tutor 
      </b>
      giải quyết giúp bạn
    </p>
    <button
      className="btn btn-primary"
      type="button"
    >
      Xem thêm
    </button>
  </div>
</div>
`;
